WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -294,18 +298,28 @@ extension SiteDesignContentCollectionViewController: UITableViewDataSource {
         }
         cell.delegate = self
         cell.selectionStyle = UITableViewCell.SelectionStyle.none
-        cell.section = isLoading ? nil : sections[indexPath.row]
-        cell.isGhostCell = isLoading
+
         if isLoading {
+            cell.section = nil
+            cell.isGhostCell = true
             cell.ghostThumbnailSize = ghostThumbnailSize
+            cell.collectionView.allowsSelection = false
+        } else {
+            let section = sections[indexPath.row]
+            cell.section = section
+            cell.isGhostCell = false
+            cell.collectionView.allowsSelection = true
+            cell.horizontalScrollOffset = sectionHorizontalOffsets[section.categorySlug] ?? .zero
```

Comment:
This same code (as well as the conformance to `CategorySectionTableViewCellDelegate`, is in both `SiteDesignContentCollectionViewController` and `FilterableCategoriesViewController`. Since they both descend from `CollapsableHeaderViewController`, I wonder if there's a way to put this code in the parent class only? Not sure if it is.